rmarkdown-workflow.Rmd

```diff
@@ -58,8 +87,12 @@ Much of the good advice about using lab notebooks effectively can also be transl
     packages available on a specified date. A quick and dirty hack is to include
     a chunk that runs `sessionInfo()` --- that won't let you easily recreate 
     your packages as they are today, but at least you'll know what they were.
+    
+*   Si quieres que tu código sea reproducible a largo plazo (esto quiere decir que        puedas regresar a ejecutarlo el mes próximo o el año próximo), necesitarás            registrar las versiones de los paquetes que tu código usa. Un enfoque riguroso es     usar __packrat__, <http://rstudio.github.io/packrat/>, el cual almacena paquetes      en tu directorio de proyecto, or __checkpoint__,                                     <https://github.com/RevolutionAnalytics/checkpoint>, el cual reinstala paquetes        disponibles en una fecha determinada .  Un truco rápido y sucio es incluir un         bloque que ejecute `sessionInfo()` --- , esto no te permitirá recrear fácilmente      tus paquetes tal y como estan hoy, pero por lo menos sabrás cuales eran.    
```

Comment:
hay un espacio extra en *determinada . *
*quick and dirty hack* es una expresión en inglés que no requiere una traducción literal. Tal vez es suficiente traducirlo como un *Un truco rápido*